web/assets/service-worker.js

```diff
@@ -0,0 +1,60 @@
+/*global console self caches fetch setTimeout clearTimeout*/
+/*
+   Network first cache strategy
+ */
+const CACHE_NAME = "v1";
+const PREFETCH_CACHE_FILES = [
+    "/",
+    "/login",
+    "/about",
+    "/static/node_modules/@fortawesome/fontawesome-free/css/all.min.css",
+    "/static/assets/css-dist/main.css",
+    "/static/assets/css/icons.css",
+    "/static/assets/ts-dist/global.bundle.js",
```

Comment:
prefetching the watch bundle and admin bundle would be nice as well, did you check wether this plays nicely with our `?version=abcde` caching strategy? :D if not we need to serve the files differently and without the version tag.

Reply:
Sorry for late reply, I do not see any issue with our current caching strategy as it is implemented network first anyways right now. So it tries on every request to first get it from the server. If that fails, it will fallback on the service-worker cache.
`e.respondWith(fromNetwork(e.request, 10000).catch(() => fromCache(e.request)));`

Did I miss something ?